Cargo.toml

```diff
@@ -1,50 +1,14 @@
-# Copyright 2019 The Fuchsia Authors
+# Copyright 2024 The Fuchsia Authors
 #
 # Licensed under a BSD-style license <LICENSE-BSD>, Apache License, Version 2.0
 # <LICENSE-APACHE or https://www.apache.org/licenses/LICENSE-2.0>, or the MIT
 # license <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your option.
 # This file may not be copied, modified, or distributed except according to
 # those terms.
 
-[package]
-name = "omaha_client"
-version = "0.1.0"
-edition = "2021"
-description = "Platform- and product-agnostic implementation of the client end of the Omaha Protocol"
-license = "BSD-2-Clause OR Apache-2.0 OR MIT"
-repository = "https://github.com/google/omaha-client"
-readme = "README.md"
-
-exclude = [".*"]
-
-[dependencies]
-anyhow = "1.0.75"
-chrono = "0.4.19"
-ecdsa =  { version = "0.14.8", features = ["pem"] }
-elliptic-curve = "0.12.3"
-futures = "0.3.19"
-hex = "0.3.2"
-http = "0.2.4"
-hyper = "0.14.19"
-itertools = "0.10.1"
-p256 = "0.11.1"
-pkcs8 = { version = "0.9.0", features = ["pem"] }
-pretty_assertions = "1.2.1"
-rand = "0.8.4"
-serde = { version = "1.0.147", features = ["derive"] }
-serde_json = "1.0.87"
-serde_repr = "0.1.7"
-sha2 = "0.10.6"
-signature = "1.6.4"
-thiserror = "1.0"
-tracing = "0.1.37"
-typed-builder = "0.10.0"
-uuid = { version = "1.1.2", features = ["serde", "v4"] }
-pin-project = "1.0.11"
-
-[dev-dependencies]
-assert_matches = "1.5.0"
-futures-timer = "0.3.0"
-hyper-rustls = { version = "0.25", features = ["http2"] }
-tokio = { version = "1", features = ["full"] }
-url = "1.7.2"
+[workspace]
+resolver = "2"
+members = [
+    "mock-omaha-server",
+    "omaha-client",
+]
```

Comment:
nit: it seems like there isn't a trailing newline here, could you add one?

Reply:
Applied in upcoming pull request.